psalm.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<psalm
```

Comment:
Why did you rename the file?

Reply:
I understand `.dist` files as generic files without specific configuration shipped with a library and once they have custom configuration you can remove that suffix. In this case this configuration file is specific for this package and it's not supposed to be used by someone else.

Anyway, I can append the `.dist` suffix if that is not right.

Reply:
Yes, please add it back.